docs/meshcloud.customer.md

```diff
@@ -30,6 +30,11 @@ You can initially setup the role in the dropdown which describes the access leve
 Press "+" to send an invitation email. This step is necessary in order to give a user access to your [projects](meshcloud.project.md).
 After this action the created invitation will appear under the pending customer user role requests section.
 
+## View User Groups
```

Comment:
should we call them "User Groups" or "meshCustomer User Groups" (which is sort of the official name)? It's clear here since we're on the meshCustomer page, but e.g. below the heading also reads "Assign meshCustomer Roles". 

@chewbeccy what do you think?

Reply:
One thing I want to raise here is that we should at least switch to user group or simply group in the description of the feature. Otherwise if we are going to write a long description with multiple paragraphs and stick with "meshCustomer User Group" I think we would be using a lot of words without adding much information.

Reply:
The official name should stay "meshCustomer User Group" because we maybe want to add project user groups or global user groups in the future. When the context is clear "Customer" its ok to reference them simply as user groups like @pubudug suggested.